config/estuary.go

```diff
@@ -71,6 +73,10 @@ func NewEstuary(appVersion string) *Estuary {
 			Disable:               false,
 			FailOnTransferFailure: false,
 			Verified:              true,
+			EnabledDealProtocolsVersions: map[protocol.ID]bool{
```

Comment:
@alvin-reyes the default is set here. If the flag is not passed, this will be used